### ISSUE TITLE
zeronet wants Python 3

### ISSUE THREAD

**Issue by lilydjwg** [archlinuxcn/repo#1462]
### 问题类型 / Type of issues

* 打包错误 / packaging error

### 受影响的软件包 / Affected packages

* zeronet

----

```
env[341453]: Error: Python 3.x is required
```

**Comment by SilverRainZ** [comment]
https://github.com/archlinuxcn/repo/pull/1536#issuecomment-613787117 issue 可以关了？

**Comment by yan12125** [comment]
A patch for Python 3.8 compatibility (https://github.com/HelloZeroNet/ZeroNet/commit/456e330854acdfbf706c601b7dcdc49501ac58ae) is not used in PKGBUILD in this repo yet. I guess @outloudvi tests with a locally-patched PKGBUILD.

**Comment by outloudvi** [comment]
Yep. I was testing with the current HEAD of HelloZeronet/Zeronet. Also, the new version seems to have [introduced many extra dependencies](https://github.com/HelloZeroNet/ZeroNet/blob/py3/requirements.txt) including `python-merkletools`, `python-base58` and so on, which requires extra packages. The `package()` function in PKGBUILD is well, but the package ZeroNet still need more changes to work.
